users/discord/App.js

```diff
@@ -15,11 +15,57 @@ export const usersData = {
   verified: null,
 };
 const urlParams = new URLSearchParams(window.location.search);
+const isDev = urlParams.get('dev') === 'true';
 
 let activeTab = urlParams.get('tab') ?? 'in_discord';
-
+const INITIAL_USERS = 10;
```

Comment:
Nit: if there is any constants file please add constants like `INITIAL_USERS` in it and then use it here.